Exiled.API/Features/Round.cs

```diff
@@ -79,6 +79,11 @@ public static bool IsLobbyLocked
         /// </summary>
         public static int ChangedIntoZombies => RoundSummary.changed_into_zombies;
 
+        /// <summary>
+        /// Gets the round.
```

Comment:
Like SebasCapo said, please make this more descriptive. "Gets the amount of rounds that have occurred since the server started." perhaps?